### PR TITLE
Add multiple sensor types for Trafikverket

### PR DESCRIPTION
**Pull request by endor-force**
**Description:**
Added precipitation instructions according to code change in sensor. Breaking change in configuration.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15935

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.trafikverket_weatherstation.markdown [hunk 1]
```diff
@@ -46,7 +47,7 @@ station:
   required: true
   type: string
 type:
-  description: Defines which temperature you want (`air` or `road`).
+  description: Defines which data you want (`air`, `road` or `precipitation`).
```
Comment (tjorim) [inline source/_components/sensor.trafikverket_weatherstation.markdown]: Maybe we could clarify that the first two are about the temperature?

**Comment by endor-force** [comment]
I will investigate possibilities to extend and use pytrafikverket lib..

File: source/_components/sensor.trafikverket_weatherstation.markdown [hunk 2]
```diff
@@ -29,31 +30,36 @@ sensor:
...
+        - road_temp
+        - humidity
+        - precipitation
+        - wind_direction
+        - wind_direction_text
+        - wind_speed
 ```
 
 {% configuration %}
-name:
```
Comment (MartinHjelmare) [inline source/_components/sensor.trafikverket_weatherstation.markdown:36]: Don't change the configuration format. See:
https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_components/sensor.trafikverket_weatherstation.markdown (hunk 2 above)
Comment (endor-force) [reply source/_components/sensor.trafikverket_weatherstation.markdown:36]: Corrected in accordance with standards.

File: source/_components/sensor.trafikverket_weatherstation.markdown [hunk 3]
```diff
@@ -29,31 +30,48 @@ sensor:
...
+api_key:  
+  description: Your personal API key from Trafikverket.  
+  required: true  
+  type: string  
+station:  
+  description: Name of the weather station from Trafikverket.  
+  required: true  
+  type: string  
+monitored_conditions: 
+  description: Conditions to display in the frontend.
```
Comment (MartinHjelmare) [inline source/_components/sensor.trafikverket_weatherstation.markdown]: This doesn't have anything to do with the frontend per se, but decides what entities will be created for the platform, ie what states to track.

Comment (endor-force) [reply source/_components/sensor.trafikverket_weatherstation.markdown]: Good input, i will clarify

**Review by MartinHjelmare — DISMISSED** [review]
Looks good! If you can add `ha_iot_class` too to the front matter that would be good. It should be `cloud polling`.

**Comment by endor-force** [comment]
Done, don't really know why it says i dismissed your review ;)

**Review by MartinHjelmare — APPROVED** [review]

**Comment by MartinHjelmare** [comment]
We've set up github to automatically do that when new changes are pushed to require a new review.

**Pull request merged**